phantom/phantomjs-shims.js

```diff
@@ -0,0 +1,35 @@
+(function() {
```

Comment:
They are not hosted anywhere and the usual approach is to copy the code to the repo.